Lib/test/test_raise.py

```diff
@@ -228,6 +228,72 @@ def test_accepts_traceback(self):
             self.fail("No exception raised")
 
 
+class TestTracebackType(unittest.TestCase):
+
+    def raiser(self):
+        raise ValueError
+
+    def test_attrs(self):
+        try:
+            self.raiser()
+        except Exception as exc:
+            tb = exc.__traceback__
+
+        assert isinstance(tb.tb_next, types.TracebackType)
```

Comment:
These assertions all need to use the unittest assert methods, not assert statements.